tests/test_grids.py

```diff
@@ -542,19 +560,20 @@ class MyParticle(ptype[mode]):
 
     lonp = 179.8
     latp = 81.35
-    pset = ParticleSet.from_list(fieldset, MyParticle, lon=lonp, lat=latp, depth=.2)
+    pset = pset_type[pset_mode]['pset'].from_list(fieldset, MyParticle, lon=lonp, lat=latp, depth=.2)
     pset.execute(pset.Kernel(sampleVel), runtime=0, dt=0)
     pset.zonal[0] = fieldset.U.units.to_source(pset.zonal[0], lonp, latp, 0)
     pset.meridional[0] = fieldset.V.units.to_source(pset.meridional[0], lonp, latp, 0)
-    assert abs(pset.zonal[0] - 1) < 1e-3
-    assert abs(pset.meridional[0]) < 1e-3
-    assert abs(pset.vertical[0] - 1) < 1e-3
+    assert (pset[0].zonal-1) < 1e-3
+    assert (pset[0].meridional-1) < 1e-3
+    assert (pset[0].vertical-1) < 1e-3
```

Comment:
Why no `abs()` here anymore?

Reply:
nicely spotted! indeed that was a bit of a rework error in the test. fixing it.